code/game/objects/structures/barsigns.dm

```diff
@@ -120,3 +120,8 @@
 	name = "Honked 'n' Loaded"
 	icon = "honkednloaded"
 	desc = "Honk."
+
+/datum/barsign/coderbus
+	name = "The Coderbus"
+	icon = "thecoderbus"
+	desc = "The Coderbus is a very controversial establishment known for its large variety of drinks."
```

Comment:
How about "known for constantly changing its drinks." ? Seems more appropriate for how often the code is changed!